smart-contract-verifier/smart-contract-verifier-http/src/verification_response.rs

```diff
@@ -3,14 +3,36 @@ use serde::{Deserialize, Serialize};
 use smart_contract_verifier::{SourcifySuccess, VerificationSuccess};
 use std::{collections::BTreeMap, fmt::Display};
 
-#[derive(Debug, Deserialize, Serialize, PartialEq, Eq)]
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq, Eq)]
 pub struct VerificationResponse {
     pub message: String,
     pub result: Option<VerificationResult>,
     pub status: VerificationStatus,
 }
 
-#[derive(Debug, Deserialize, Serialize, PartialEq, Eq)]
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq, Eq)]
+#[serde(tag = "type")]
```

Comment:
We have a test here
https://github.com/blockscout/blockscout-rs/pull/205/files#diff-c91c0999946402b4fe047f1e433609c4274379e374ba0c17628175e410e65307R215

You are right - it would be encoded as `{"type": "Main"}`